kratos/spatial_containers/bounding_volume_tree.h

```diff
@@ -270,11 +270,11 @@ class _6DOP : public kDOP // AABB
     KRATOS_CLASS_POINTER_DEFINITION(_6DOP);
     _6DOP() : kDOP() {this->Initialize();}
     virtual ~_6DOP() {}
-    virtual std::size_t NumberOfDirections() const {return 3;}
+    virtual std::size_t NumberOfDirections() const override {return 3;}
 
 private:
     static const double msDirection[][3];
-    virtual const double (*Direction() const)[3];
+    virtual const double (*Direction() const override)[3];
```

Comment:
I hope @pooyan-dadvand will help us on this. I just put 'override' everywhere gcc was recommending it (the flag -Wsuggest-override).